.travis.yml

```diff
@@ -1,7 +1,6 @@
 # Travis CI script
 # This is a skeleton created by zproject.
 # You can add hand-written code here.
-### Note: Customized after generation; take care upon regens
```

Comment:
Why is this and lower comments about customizations removed? Customizations themselves are still needed and still here...